spark-hello-world/Configure-Hadoop.ps1

```diff
@@ -0,0 +1,56 @@
+<#
+.SYNOPSIS
+Downloads the 'winutils' repo needed to run Hadoop on Windows, as well as configures the HADOOP_HOME environment 
+variable.
+
+.DESCRIPTION
+This function checks if the winutils folder exists and if it does not, clones the winutils repository from GitHub.
+It then determines the absolute path to the highest versioned subfolder within the winutils folder and sets the
+HADOOP_HOME environment variable to this path. If the HADOOP_HOME environment variable does not exist, it is created.
+
+.PARAMETER None
+
+.EXAMPLE
+Assuming you trust this script, you will need to run the script via an ELEVATED command line
+and possibly bypass the default Powershell execution policy.  Here are the command line steps below:
+cd <same-directory-as-this-script>
+powershell.exe -ExecutionPolicy Bypass -File Configure-Hadoop.ps1
+
+#>
+
+$currentPrincipal = New-Object Security.Principal.WindowsPrincipal([Security.Principal.WindowsIdentity]::GetCurrent())
+if (-Not $currentPrincipal.IsInRole([Security.Principal.WindowsBuiltInRole]::Administrator)) {
+    # otherwise we are done
+    Write-Host "You are not an admin and this script requires elevated permissions.  Please execute this script as an admin..."
+    return
+}
+
+# If we don't find the 'winutils' directory, assume it hasnt been setup yet and proceed.
+$winutilsPath = "$PSScriptRoot\winutils"
+if (Test-Path $winutilsPath) {
+    # otherwise we are done
+    Write-Host "Winutils folder already exists, exiting..."
+    return
+}
+
+Write-Host "Cloning winutils repository..."
+git clone https://github.com/steveloughran/winutils $winutilsPath
+
+# find the highest versioned hadoop install in winutils
+$hadoopHome = Get-ChildItem -Path $winutilsPath -Directory |
```

Comment:
I think we actually use 2.8.3, not the latest... although that will probably change when we upgrade spark. Maybe just code it to be 2.8.3 for now?